pom.xml

```diff
@@ -128,6 +130,44 @@
             </activation>
             <properties>
                 <cmake.cflags>-m32</cmake.cflags>
+                <configured.classifier>linux-x86_32</configured.classifier>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>windows-64</id>
+            <properties>
+                <configured.classifier>windows-x86_64</configured.classifier>
+            </properties>
+        </profile>
+        <profile>
+            <id>windows-32</id>
+            <properties>
+                <configured.classifier>windows-x86_32</configured.classifier>
+            </properties>
+        </profile>
+        <profile>
+            <id>linux-armv6</id>
+            <properties>
+                <configured.classifier>linux-armv6</configured.classifier>
```

Comment:
you said we're matching the os-maven-plugin, but it just has "arm_32" and "aarch64"

we want our stuff to work with this plugin

if we need different armv6/armv7 builds, should the "arm_32" just include 2 different binaries?